plugins/nodes/src/js/columns/NodesTablePublicIPColumn.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { Tooltip } from "reactjs-components";
+import { TextCell } from "@dcos/ui-kit";
+import { Trans } from "@lingui/macro";
+
+import Node from "src/js/structs/Node";
+
+const NodesTablePublicIpColumn = (item: Node) => {
```

Comment:
Just random nit-picking:

```suggestion
export default (item: Node) => {
```

or 
```suggestion
export default function NodesTablePublicIpColumn(item: Node) {
```

Reply:
We decided that we want to have a single export at the end of the file as a new convention. Which makes these comments outdated.

Reply:
Ah okay, I might have forgot about this